smarts/core/utils/file.py

```diff
@@ -94,3 +95,17 @@ def smarts_log_dir() -> str:
 
 def make_dir_in_smarts_log_dir(dir):
     return os.path.join(smarts_log_dir(), dir)
+
+
+@contextmanager
+def supress_pkg_resources():
+    import sys
+
+    import pkg_resources
+
+    from .invalid import raise_import_error
```

Comment:
Perhaps use full path instead of relative path import?

```suggestion
    from smarts.core.utils.invalid import raise_import_error
```